dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/PowerSystemStabilizers/Standard/PssIEEE2B.mo

```diff
@@ -0,0 +1,150 @@
+//  parameter Types.PerUnit Ks4 "Gain on signal #2 input after ramp-tracking filter.  Typical value = 1";
+  parameter Types.ApparentPowerModule SNom "Nominal apparent power in MVA";
+  parameter Types.Time t1 "Lead time constant of first lead/lag (>= 0). Typical value = 0,12";
+  parameter Types.Time t2 "Lag time constant of first lead/lag (>= 0). Typical value = 0,02";
+  parameter Types.Time t3 "Lead time constant of second lead/lag (>= 0). Typical value = 0,3";
+  parameter Types.Time t4 "Lag time constant of second lead/lag (>= 0). Typical value = 0,02";
+  parameter Types.Time t6 "Time constant on signal #1 (>= 0). Typical value = 0";
+  parameter Types.Time t7 "Time constant on signal #2 (>= 0). Typical value = 2";
+  parameter Types.Time t8 "Lead of ramp tracking filter (>= 0). Typical value = 0,2";
+  parameter Types.Time t9 "Lag of ramp tracking filter (>= 0). Typical value = 0,1";
+  parameter Types.Time t10 "Lead time constant of third lead/lag (>= 0). Typical value = 0";
+  parameter Types.Time t11 "Lag time constant of third lead/lag (>= 0). Typical value = 0";
+  parameter Types.Time tw1 "First washout on signal #1 (>= 0). Typical value = 2";
+  parameter Types.Time tw2 "Second washout on signal #1 (>= 0). Typical value = 2";
+  parameter Types.Time tw3 "First washout on signal #2 (>= 0). Typical value = 2";
+  parameter Types.Time tw4 "Second washout on signal #2 (>= 0). Typical value = 0";
+  parameter Types.VoltageModulePu Vsi1Max "Input signal #1 maximum limit (> vsi1min). Typical value = 2";
+  parameter Types.VoltageModulePu Vsi1Min "Input signal #1 minimum limit (< vsi1max). Typical value = -2";
+  parameter Types.VoltageModulePu Vsi2Max "Input signal #2 maximum limit (> vsi2min). Typical value = 2";
+  parameter Types.VoltageModulePu Vsi2Min "Input signal #2 minimum limit (< vsi2max). Typical value = -2";
+  parameter Types.VoltageModulePu VstMax "Stabilizer output maximum limit (> vstmin). Typical value = 0,1";
+  parameter Types.VoltageModulePu VstMin "Stabilizer output minimum limit (< vstmax). Typical value = -0,1";
+//  parameter Types.ActivePower PNomAlt "Nominal active (alternator) power in MW";
+//  parameter Types.VoltageModulePu Upss0Pu "Initial voltage output in pu (base UNom)";
+
+  //Input variables
+  Modelica.Blocks.Interfaces.RealInput PGenPu(start = PGen0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-254, -40}, extent = {{-14, -14}, {14, 14}}, rotation = 0), iconTransformation(origin = {-120, -60}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput omegaPu(start = SystemBase.omega0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-254, 40}, extent = {{-14, -14}, {14, 14}}, rotation = 0), iconTransformation(origin = {-120, 60}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+
+  //Output variable
+  Modelica.Blocks.Interfaces.RealOutput UpssPu(start = Upss0Pu) annotation(
```

Comment:
Description ?

Moreover, it ought to be UPssPu according to the camel-case convention.